### PR TITLE
Create data model

### PR DESCRIPTION
**Pull request by emmiiorji**
This change creates data models for the application following the [database design](https://camo.githubusercontent.com/28ca8105e51cc8f2c4599399c2988228c6804e52baa366d5c11fa2087130f3d9/68747470733a2f2f7265732e636c6f7564696e6172792e636f6d2f656d6d69692f696d6167652f75706c6f61642f76313636333136353630342f67656e6572616c2f626c6f675f6170705f6572645f686f726e39742e706e67)

The following apply:

- The project is set up with Postgres database.
- Table and column names match the ERD diagram.
- Foreign keys are included.
- All columns that are foreign keys have corresponding indices.

### CODE REVIEW

File: db/migrate/20220914181032_add_post_counter_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddPostCounterToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :post_counter, :integer
+  end
```
Comment (brytebee) [inline db/migrate/20220914181032_add_post_counter_to_users.rb:4]: 👏👏👏👏👏 Awesome way to add missed columns/attributes to a class/model.

**Review by brytebee — APPROVED** [review]
Hi, @emmiiorji👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great schema design.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by julie-ify** [comment]
Hi @emmiiorji,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — COMMENTED** [review]
## Status: Approved :heavy_check_mark:

Hi @emmiiorji 

Your project is complete! 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request ✔️  
- No linter errors✔️ 


-- [optional]great job updating to the latest readme. when you merge ensure that the main branchs README is also updated like that of this feature branch.
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**